### PR TITLE
[Docs] Add new Microservices item with nested_options

### PR DESCRIPTION
**Pull request by eperinan**
This PR is related to https://github.com/frees-io/freestyle-rpc/issues/176.

In this PR, I am adding a new main section in the left menu called `Microservices` with their nested options. All these new content will be available in the https://github.com/frees-io/freestyle-rpc repository.

Apart from this, I created a new simple layout in the path `/docs/libraries/` to link the libraries available.

### CODE REVIEW

**Comment by eperinan** [comment]
Does it make sense @juanpedromoreno?

File: docs/src/main/resources/microsite/data/menu.yml [hunk 1]
```diff
@@ -109,13 +109,45 @@ options:
...
 
-      - title: Microservices
-        url: docs/rpc
+      - title: Quickstart
+        url: docs/rpc/quickstart
+
+      - title: Core concepts
+        url: docs/rpc/core-concepts
+
+      - title: Implementations
```
Comment (juanpedromoreno) [inline docs/src/main/resources/microsite/data/menu.yml]: Related to https://github.com/frees-io/freestyle-rpc/pull/190#discussion_r172783943

File: docs/src/main/tut/docs/libraries/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
```
Comment (juanpedromoreno) [inline docs/src/main/tut/docs/libraries/README.md:1]: We might need a similar document like this for the `docs/rpc/README.md` path, where all the subsections above are included.

Comment (eperinan) [reply docs/src/main/tut/docs/libraries/README.md:1]: I have one like this in RPC as well. Tonight I think that I will push the commit. Thanks for the reminder anyway.

Comment (eperinan) [reply docs/src/main/tut/docs/libraries/README.md:1]: Pushed content : https://github.com/frees-io/freestyle-rpc/blob/35ebd55bb43fae28dcf927a65e438858f994ae51/docs/src/main/tut/README.md

File: docs/src/main/resources/microsite/data/menu.yml [hunk 3]
```diff
@@ -109,13 +109,42 @@ options:
...
 
-      - title: Microservices
-        url: docs/rpc
+      - title: Quickstart
+        url: docs/rpc/quickstart
+
+      - title: Core concepts
+        url: docs/rpc/core-concepts
+
+      - title: IDL Generation
```
Comment (juanpedromoreno) [inline docs/src/main/resources/microsite/data/menu.yml]: Let's sort all these menu items as they are here https://github.com/frees-io/freestyle-rpc/blob/master/docs/src/main/tut/README.md

Thanks

**Review by juanpedromoreno — APPROVED** [review]
Once menu is sorted, LGTM, thanks!

**Pull request merged**